test/action_controller/json_api/linked_test.rb

```diff
@@ -39,70 +38,76 @@ def setup_post
 
           def render_resource_without_include
             setup_post
-            render jsonapi: @post
+            render json: @post
```

Comment:
Using it here feels like a mix of concerns. The `jsonapi` renderer is opt-in and this test doesn't directly concern the behavior of the `jsonapi` renderer.